sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -400,20 +400,46 @@ es3fLifetimeTests.TfDeleteActiveTest = function(name, description) {
 setParentClass(es3fLifetimeTests.TfDeleteActiveTest, tcuTestCase.DeqpTest);
 
 es3fLifetimeTests.TfDeleteActiveTest.prototype.iterate = function() {
+/** @const */ var s_xfbVertexSource =
+    '#version 300 es\n' +
+    'void main ()\n' +
+    '{\n' +
+    ' gl_Position = vec4(float(1) / 2.0, float(1 % 2) / 2.0, 0.0, 1.0);\n' +
```

Comment:
This deviates from c++ code.  I didn't use gl_VertexID here. Chrome doesn't support gl_VertexID yet and gl_VertexID doesn't influence the test result in this case.